packages/cli/src/evaluation/test-runner/test-runner.service.ee.ts

```diff
@@ -267,6 +273,9 @@ export class TestRunnerService {
 			} catch (e) {
 				// In case of an unexpected error, increment the failed count and continue with the next test case
 				await this.testRunRepository.incrementFailed(testRun.id);
+
+				// Report error to Sentry
+				captureException(e);
```

Comment:
We have [`ErrorReporter`](https://github.com/n8n-io/n8n/blob/master/packages/core/src/error-reporter.ts#L0-L1) that we've been using for reporting errors. Let's use that so we don't have to depend on sentry directly. So this would be `errorReporter.report(e)`

Reply:
Thanks, I swapped it with `errorReporter.error` - this will also wrap an exception into proper class.